pdns/recursordist/Makefile.am

```diff
@@ -369,6 +371,9 @@ rec_control_SOURCES = \
 	rec_control.cc \
 	unix_utility.cc
 
+rec_control_LDADD = \
```

Comment:
I don't see why we'd need libcap support in `rec_control` (apart from "this is `#include`d because reasons")

Reply:
Because it links `misc.o` in, which now refers to symbols from `libcap`.

Reply:
Fixed!